demisto_sdk/commands/common/tests/reputation_test.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+from demisto_sdk.commands.common.hook_validations.reputation import ReputationValidator
+from demisto_sdk.commands.common.hook_validations.structure import StructureValidator
+
+
+class TestReputationValidator:
```

Comment:
there is no meaning for the class, usually test classes are use to gather different use cases of same area (for example, multiple tests per function: sanity, special params, exceptions...)